ui/ducks/send/send.js

```diff
@@ -1638,6 +1707,8 @@ export function useMyAccountsForRecipientSearch() {
  */
 export function updateRecipient({ address, nickname }) {
   return async (dispatch, getState) => {
+    // Do not addHistoryEntry here as this is called from a number of places
```

Comment:
Because the *way* the recipient is added is important and not known here, so in each place this is used in the code I have added logging that specifies how/why the recipient is updated.

Reply:
I'll update the comment to be more specific... Alternatively I could create variations of updateRecipient with specifics to be used downstream.

Reply:
oh, yes, I see
I hadn't seen the changes in pages/send/send.js before